imagescan/scanner.go

```diff
@@ -71,6 +72,9 @@ func (s *Scanner) ScanImage(ctx context.Context, params ScanImageParams) (rerr e
 	if params.ContainerRuntime == "" {
 		return errors.New("container runtime is required")
 	}
+	if params.Mode == "" {
```

Comment:
In line 98 there is check for empty mode, now it is never true and can be removed